src/session/LogoutSession.js

```diff
@@ -9,29 +9,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
 import * as actions from './sessionActions';
-import ifAuthenticated from '../util/ifAuthenticated';
+import polyglot from '../i18n';
 
 export class LogoutSession extends React.Component {
-  componentWillMount() {
-    const { localLogout, authenticated, localIfAuthenticated } = this.props;
-    localIfAuthenticated(authenticated, localLogout);
+  constructor() {
+    super();
+    this.state = {
+      logoutClicked: false,
+    };
+  }
+
+  handleLogoutClick = (federated = undefined) => {
+    this.setState({ logoutClicked: true });
+    this.props.localLogout(federated);
   }
 
   render() {
-    return null;
+    const { authenticated } = this.props;
+    if (!authenticated && !this.state.logoutClicked) {
+      return <Redirect to="/forbidden" />;
+    }
+
+    return (
+      <div className="one-column one-column--narrow logout-container">
+        <button className="button--primary-outline cta-link--block" onClick={() => this.handleLogoutClick()}>{polyglot.t('logoutSession.logutLearningpathSite')}</button>
+        <b>{polyglot.t('logoutSession.or')}</b>
```

Comment:
Hvorfor ville du valgt strong? Strong sier noen om hvor viktig teksten er for eksempel og jeg føler helt klart at "eller" ikke er viktig i seg selv.